venia-integration-tests/run-tests.js

```diff
@@ -78,20 +78,25 @@ const port = new URL(baseUrl).port;
 
 let dockerCommand = null;
 
+let dockerImage = "cypress/included:8.3.1";
+if (process.env.CI) {
+    dockerImage = "docker-hub-remote.dr-uw2.adobeitc.com/" + dockerImage;
```

Comment:
Will this address ever change? Does this file have access to the same environment variable `$JFROG_REPO` like the cicd scripts?